File: ReptiCare/Length+CoreDataProperties.m
//
//  Length+CoreDataProperties.m
//  ReptiCare
//
//  Created by Kevin De Koninck on 24/09/2016.
//  Copyright © 2016 Kevin De Koninck. All rights reserved.
//

#import "Length+CoreDataProperties.h"

@implementation Length (CoreDataProperties)

+ (NSFetchRequest<Length *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"Length"];
}

@dynamic date;
@dynamic length;
@dynamic uniqueID;
@dynamic reptile;

@end

File: ReptiCare/Length+CoreDataProperties.swift
//
//  Length+CoreDataProperties.swift
//  ReptiCare
//
//  Created by Kevin De Koninck on 24/09/2016.
//  Copyright © 2016 Kevin De Koninck. All rights reserved.
//

import Foundation
import CoreData 

extension Length {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Length> {
        return NSFetchRequest<Length>(entityName: "Length");
    }

    @NSManaged public var date: NSDate?
    @NSManaged public var length: NSNumber?
    @NSManaged public var uniqueID: NSNumber?
    @NSManaged public var reptile: Reptile?

}
